cmake/Dependencies.cmake

```diff
@@ -90,6 +90,9 @@ if(USE_XPU)
     message(WARNING "Not compiling with XPU. Could NOT find SYCL."
     "Suppress this warning with -DUSE_XPU=OFF.")
     caffe2_update_option(USE_XPU OFF)
+  else()
+    # Workaround about XPU ABI neutral build
+    string(APPEND CMAKE_CXX_FLAGS " -D__INTEL_PREVIEW_BREAKING_CHANGES")
```

Comment:
Please get commitment from the compiler team.

Reply:
Can you, please, also get more details into PR description? what "ABI neutral build" means? what is sycl-preview library? will ever changes from preview available in stock?